app/javascript/utils/use-storage.ts

```diff
@@ -1,4 +1,4 @@
-import React, { useCallback, useState } from 'react'
+import React, { useState } from 'react'
```

Comment:
This handles saving to local storage. We need to encode / decode the objects to JSON as localstorage only accepts strings.